### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lokathor**
* Change the time thing so that it's just a google link to the local time conversion. Should be more error-proof.
* Give the hangouts link directly
* Also link the community discord
* Also link the reddit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,11 @@ If you're interested in learning more about our governance or becoming a member
 
 ## Join the fun!
 
-We have a meeting every other Wednesday at **19:00 UTC**.  This is 13:00 PST (UTC-7), 16:00 US EST (UTC-4), 20:00 BST (UTC+1), and 04:00 on Thursday JST (UTC+9): <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWgxZnN2cWR2NjBhZGlhODBwNXRoc3RrMDBfMjAxOTA3MTBUMTkwMDAwWiAxM3JhM3R1ZXFrcWJmZnBmMmc1NmZvMmN0c0Bn&amp;tmsrc=13ra3tueqkqbffpf2g56fo2cts%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>    
-We also have a [discord channel][discord].
+* We have a [Google Hangouts](https://hangouts.google.com/call/BgkpIXkZghZH92NqjNWsAEEI) meeting every other Wednesday at **19:00 UTC**.
+* [This link](https://www.google.com/search?q=1900+utc+in+local+time) should convert that time to your local time zone's time.
+* We have a `wg-gamedev` channel in the [Offical Rust Discord][discord].
+* There's a `game-and-graphics` channel in the [Community Rust Discord](https://discordapp.com/invite/aVESxV8).
+* There's a [Rust Gamedev](https://www.reddit.com/r/rust_gamedev/) subreddit as well.
```
Comment (AlexEne) [inline README.md]: I'd remove this reddit link for now as it's not related to the working group and we basically have no control what's posted in there.

Comment (AlexEne) [reply README.md]: Actually I have a similar comment about the community discord

File: README.md [hunk 2]
```diff
@@ -27,8 +27,11 @@ If you're interested in learning more about our governance or becoming a member
 
 ## Join the fun!
 
-We have a meeting every other Wednesday at **19:00 UTC**.  This is 13:00 PST (UTC-7), 16:00 US EST (UTC-4), 20:00 BST (UTC+1), and 04:00 on Thursday JST (UTC+9): <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWgxZnN2cWR2NjBhZGlhODBwNXRoc3RrMDBfMjAxOTA3MTBUMTkwMDAwWiAxM3JhM3R1ZXFrcWJmZnBmMmc1NmZvMmN0c0Bn&amp;tmsrc=13ra3tueqkqbffpf2g56fo2cts%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>    
-We also have a [discord channel][discord].
+* We have a [Google Hangouts](https://hangouts.google.com/call/BgkpIXkZghZH92NqjNWsAEEI) meeting every other Wednesday at **19:00 UTC**.
```
Comment (AlexEne) [inline README.md]: I noticed you deleted the google calendar link and I think it's useful for people to be able to add it to their calendar. I like the reminders. Maybe add the hangouts link separately if you really want ?

Comment (Lokathor) [reply README.md]: Ah, sure. I can fix that. The date was old so I thought the Hangouts part of it was the only important part left.

File: README.md (hunk 1 above)
Comment (Lokathor) [reply README.md]: That's an interesting question: If we're not willing to tell people about places to coordinate and interact... what are we doing at all?

* While we don't have direct moderation power over the rust_gamedev reddit, it's a place where many serious game projects (ggez, nalgebra, etc) post information and updates. As far as I've ever seen the rust_gamedev reddit upholds with the Rust Code of Conduct that applies to the official rust discord, official rust forums, main rust reddit, and so on.
* Similarly, people chatting for too long about "not official WG business" in the `wg-gamedev` channel on the Official discord are often told that their conversation is off-topic and they should try the Community Discord for long general gamedev discussions. Also, the Community Discord explicitly holds to the Rust Code of Conduct and in fact the most active moderator of the Community Discord is also a moderator of the Official Discord and the Amethyst Discord, so there's good consistency of policy enforcement across the various Rust places.

I would certainly be happy to add an additional disclaimer that we of the WG don't moderate those spaces, but either they should be mentioned as places to go OR the `wg-gamedev` channel on the Official Discord needs to be opened up for general use as a discussion place for game development.

We need to have _some_ suggested place for people to have general conversations outside of continually focused issue discussion.

Comment (kvark) [reply README.md]: I find the links useful given an explicit disclaimer that we don't manage these platforms

Comment (Lokathor) [reply README.md]: Fixed in https://github.com/rust-gamedev/wg/pull/37/commits/55bfb47a0c6fc11cd249b9f21e13721dba0322f4

**Comment by Lokathor** [comment]
Sorry to throw in a commit just after you approve the changes hecrj, but the notification reminded me that I still hadn't made the change to explain that we don't moderate some of the places mentioned in the links.

We should be ready to go at this point.

**Pull request merged**